packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts

```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
 ) =>
     css`
     ${display("inline-flex")} :host {
-        background: transparent;
-        box-sizing: border-box;
-        font-family: ${bodyFont};
-        font-size: ${typeRampBaseFontSize};
-        fill: currentColor;
-        line-height: ${typeRampBaseLineHeight};
-        min-width: calc(${heightNumber} * 1px);
-        outline: none;
-        color: ${neutralForegroundRest}
+      background: transparent;
+      box-sizing: border-box;
+      font-family: ${bodyFont};
+      font-size: ${typeRampBaseFontSize};
+      fill: currentcolor;
+      line-height: ${typeRampBaseLineHeight};
+      min-width: calc(${heightNumber} * 1px);
+      border-radius: calc(${controlCornerRadius} * 1px);
+      outline: none;
+      color: ${neutralForegroundRest}
     }
-
     .listitem {
-        display: flex;
-        align-items: center;
-        width: max-content;
+      display: flex;
+      align-items: center;
+      border-radius: inherit;
     }
-
     .separator {
-        margin: 0 6px;
+        margin: 0 20px;
         display: flex;
-    }
-
+      }
     .control {
-        align-items: center;
-        box-sizing: border-box;
-        color: ${accentForegroundRest};
-        cursor: pointer;
-        display: flex;
-        fill: inherit;
-        outline: none;
-        text-decoration: none;
-        white-space: nowrap;
+      position: relative;
+      align-items: center;
+      box-sizing: border-box;
+      background: transparent;
+      border: calc(${strokeWidth} * 1px) solid transparent;
+      color: ${neutralForegroundRest};
+      cursor: pointer;
+      display: flex;
+      fill: inherit;
+      outline: none;
+      text-decoration: none;
+      white-space: nowrap;
+      border-radius: inherit;
+      padding: 4px 8px;
     }
-
     .control:hover {
-        color: ${accentForegroundHover};
+      color: ${neutralForegroundHover};
     }
-
     .control:active {
-        color: ${accentForegroundActive};
+      background: ${neutralFillStealthActive};
+      color: ${neutralForegroundActive};
+    }
+    .control:${focusVisible} {
+      border-color: ${focusStrokeOuter};
+      box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${focusStrokeOuter} inset;
+      color: ${neutralForegroundFocus};
     }
-
     .control .content {
-        position: relative;
+      position: relative;
     }
-
     .control .content::before {
-        content: "";
-        display: block;
-        height: calc(${strokeWidth} * 1px);
-        left: 0;
-        position: absolute;
-        right: 0;
-        top: calc(1em + 4px);
-        width: 100%;
+      content: "";
```

Comment:
Is this the link underline? Can we use text-decoration instead?

Reply:
switched to use text decoration